CraftTweaker2-API/src/main/java/crafttweaker/api/data/DataInt.java

```diff
@@ -17,7 +17,7 @@ public DataInt(int value) {
     
     @Override
     public boolean asBool() {
-        throw new RuntimeException("Cannot cast an int to a bool");
+        return value != 0;
```

Comment:
I feel this should better mimic how Java does it, which is "if this isn't true (or 1), then it should be considered false).

So change this to `value == 1`